orchagent/orchdaemon.cpp

```diff
@@ -68,6 +70,7 @@ OrchDaemon::~OrchDaemon()
     for(; it != m_orchList.rend(); ++it) {
         delete(*it);
     }
+    delete m_select;
```

Comment:
same as previous what is this change for ?

Reply:
ok got it. we have moved to base class for select object initialization

Reply:
Yes, I had to move it to base class since it's used for npu and fabric too.